travis/version-wdl-runtimes.sh

```diff
@@ -7,5 +7,11 @@ OLD_TAG="quay.io/broadinstitute/viral-ngs"
 echo Replacing $OLD_TAG with $NEW_TAG in all task WDL files
 sed -i -- "s|$OLD_TAG|$NEW_TAG|g" pipes/WDL/workflows/tasks/*.wdl
 
+CURENT_VERSION=`travis/list-docker-tags.sh | tail -1 | cut -d\: -f2`
```

Comment:
Not functionally important, but just to avoid another cosmetic PR later (also in line 13&14 below)
```suggestion
CURRENT_VERSION=`travis/list-docker-tags.sh | tail -1 | cut -d\: -f2`
```

Reply:
Oh also, maybe just to avoid multiple invocations of `list-docker-tags.sh`, could we make this more like
```
CURRENT_VERSION=`echo $NEW_TAG | cut -d\: -f2`
```

Reply:
Thanks for catching the typo!